web_server/src/lib.rs

```diff
@@ -172,6 +181,20 @@ impl WebServerDataSource for WebServerState {
 
         Ok(())
     }
+
+    fn post_staketable(&mut self, key: Vec<u8>) -> Result<(), Error> {
+        // KALEY TODO: need security checks here
```

Comment:
Should we have an issue for this?

Reply:
Good call- https://github.com/EspressoSystems/HotShot/issues/1108